molecule/default/vars/vars.yml

```diff
@@ -8,3 +8,7 @@ tas_single_node_cockpit:
   enabled: false
 tas_single_node_registry_username: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_USERNAME') }}"
 tas_single_node_registry_password: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_PASSWORD') }}"
+private_key_file_path: ~/.ssh/id_rsa
+image_id: ami-0cd93da2da7f10cab
+instance_type: m5.xlarge
+aws_key_name: CI-TEST
```

Comment:
These variables should go to `molecule/default/molecule.yml` to the `platforms` config (and then you can access them via `molecule_yml.platforms`)